A/Agama/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+using BinaryBuilder, Pkg
+
+name = "Agama"
+repo = "https://github.com/GalacticDynamics-Oxford/Agama.git"
+
+version = v"1.0"
+sources = [
+    GitSource(repo, "f392b9d6e59269d451b37b3ada2508a44bd3a80c"),
+]
+
+
+all_platforms = supported_platforms()
```

Comment:
```julia
supported_platforms(; exclude=Sys.iswindows)
```
is simpler than all of this.

Reply:
Ah ok. I also need to exclude the `musl` linux, can I do that in a similar way? Or would I need to write a little anon-func for that keyword?